contracts/modules/dispute/DisputeModule.sol

```diff
@@ -21,7 +22,7 @@ import { ShortStringOps } from "../../utils/ShortStringOps.sol";
 contract DisputeModule is
     IDisputeModule,
     BaseModule,
-    GovernableUpgradeable,
+    AccessManagedUpgradeable,
```

Comment:
Name again,  need to be consistent

Reply:
This one must be AccessManaged. AcessManager is the deployed singleton